server/community.go

```diff
@@ -32,3 +33,18 @@ func (s *Server) addHacktoberfestLabel(ctx context.Context, pr *model.PullReques
 		return
 	}
 }
+
+func (s *Server) postPRWelcomeMessage(ctx context.Context, pr *model.PullRequest) {
+	// Only post welcome Message for community member
+	if s.IsOrgMember(pr.Username) {
+		return
+	}
+
+	msg := strings.ReplaceAll(s.Config.PRWelcomeMessage, "USERNAME", "@"+pr.Username)
```

Comment:
I know we use "USERNAME" as a convention and just replace it. But it will be better if we can use `text/template`s for this. Would you be open to making that change here: https://github.com/mattermost/platform-private/pull/284? We can open more HW tickets changing the rest of the occurences to use `text/template`s

Reply:
While I see that using `text/template` is the "right" thing to do, I'm leaning toward consistency here. 1/5 that I would like to changes all occurrences of  `USERNAME` at the same time.

Reply:
Ok.

Reply:
I'm OK using this with `strings.Replace()` for the consistency. However, as a nice improvement, I'd like to see all of the `xMessage` in the `config.json` to be moved into something like `messages.go` or in a text file as an asset.

Reply:
Moved to `text/template` in [`6177372` (#246)](https://github.com/mattermost/mattermost-mattermod/pull/246/commits/617737219cb8c8a3183e0907170a6b69dabd9864)